app/contracts/authentication/omniauth_auth_hash_contract.rb

```diff
@@ -49,6 +49,7 @@ def validate_auth_hash
     end
 
     def validate_auth_hash_not_expired
+      return if auth_hash.nil?
       return unless auth_hash["timestamp"]
```

Comment:
```suggestion
      return unless auth_hash&.[]("timestamp")
```

Reply:
or perhaps this reads nicer:

```suggestion
      return unless auth_hash&.dig("timestamp")
```